<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DarkForm</class>
 <widget class="QWidget" name="DarkForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <widget class="QLCDNumber" name="lcdNumberMH">
        <property name="digitCount">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 72pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLCDNumber" name="lcdNumberHH">
        <property name="digitCount">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLCDNumber" name="lcdNumberML">
        <property name="digitCount">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcdNumberHL">
        <property name="digitCount">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLCDNumber" name="lcdNumberSH">
        <property name="digitCount">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLCDNumber" name="lcdNumberSL">
        <property name="digitCount">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 72pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
